studio/s3_provider.py

```diff
@@ -26,12 +19,22 @@ def __init__(self, config, blocking_auth=True, verbose=10, store=None):
             store)
 
     def _get(self, key, shallow=False):
-        response = self.meta_store.client.get_object(
-            Bucket=self.bucket,
-            Key=key)
+        try:
+            response = self.meta_store.client.get_object(
+                Bucket=self.bucket,
+                Key=key)
+        except self.meta_store.client.exceptions.NoSuchKey as e:
+            logging.error("Failed to retrieve key %s from %s", key, self.bucket)
```

Comment:
this is a fairly routine situation - trying to access the key and realizing it does not exsit (esp. given that we have no separate method for detecting if key is there), so maybe do logger.info / logger.debug instead in order not to blow up the output